petJournal/petJournal/Features/Auth/Scenes/Home/View/SideMenuView.swift

```diff
@@ -9,11 +9,13 @@ import SwiftUI
 
 struct SideMenuView: View {
     
-    @State private var showMenu = false
+    @State  var showMenu = false
     @StateObject var vm = SessionManager()
```

Comment:
Nome vm é muito curto, seria melhor alguma coisa mais descritiva como sessionManager mesmo
```suggestion
    @StateObject var sessionManager = SessionManager()
```

Reply:
Nomenclatura corrigida.